package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "phantom-backend-pirates",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha './test/test.js'",
+    "start": "node app.js",
+    "devServer": "nodemon --exec node app.js",
+    "lint": "eslint . --ext .js --ext .jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlp-rwanda/Phantom-Backend-Pirates.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atlp-rwanda/Phantom-Backend-Pirates/issues"
+  },
+  "homepage": "https://github.com/atlp-rwanda/Phantom-Backend-Pirates#readme",
+  "dependencies": {
+    "bcrypt": "^5.0.1",
+    "chai": "^4.3.6",
+    "debug": "^4.3.3",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.2",
+    "http-errors": "^2.0.0",
+    "jsonwebtoken": "^8.5.1",
+    "mocha": "^9.2.0",
+    "morgan": "^1.10.0",
+    "sinon": "^13.0.1",
```

Comment:
Please installing packages is not enough, consider making a full configuration to make sure testing is working